### DISCUSSION TITLE
dotnet showing core version in framework project folder

### DISCUSSION THREAD

**Discussion by PilotBob** [JanDeDobbeleer/oh-my-posh#1564] [Troubleshoot]
Perhaps this is a feature request. But currently the dotnet segment seems to display the .net core version even if it is in a folder with a .net framework project. 

Is there some setting I am missing? 

If not, it would be nice to enhance this to show the .Net framework version. Not 100% sure how since the .sln I don't think specifies the framework being used. However, legacy csproj does have a TargetFrameworkVersion element which the SDK projects have a TargetFarmework element.

**Comment by JanDeDobbeleer** [comment]
yes, it only supports dotnet core. So indeed, that's a feature request :-)
